app/models/host_aspect_base.rb

```diff
@@ -0,0 +1,25 @@
+class HostAspectBase < ActiveRecord::Base
```

Comment:
> > Might make sense to rename HostAspect, as it's causing confusion.
> 
> I am not too attached to this name, suggestions?

Agreed, it's too ambiguous.  Perhaps the base class should be a plain "Aspect", which leaves the association model being a HostAspect, which makes a little more sense.